.github/workflows/ci.yml

```
@@ -541,6 +541,87 @@ jobs:
             --require-commit "$GITHUB_SHA" \
             | tee -a "$GITHUB_STEP_SUMMARY"
 
+  # Nine sugarbin shell contracts used to be reachable only through
+  # test-rust, which is deliberately outside per-tip CI. Run the exact shell
+  # population directly: one job, three batches, and one attendance receipt.
+  sugarbin-shell-contracts:
+    name: sugarbin shell contracts (9/9 attendance)
+    runs-on: [self-hosted, Linux, X64]
+    timeout-minutes: 15
+    env:
+      SUGARBIN_SHELL_TIER_ROOT: ${{ runner.temp }}/sugarbin-shell-tier
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)